Sito/back/home_admin.php

```diff
@@ -61,9 +67,10 @@
                                         <dd><input type=\"button\" name=\"elimina\" value=\"Elimina\"/></dd>
                                     ";
                 }
+                $paginaHTML = str_replace('<formNews />', $formNews, $paginaHTML);
                 $paginaHTML = str_replace('<messaggio />', $erroriNews, $paginaHTML);
                 echo str_replace('<notizie />', $notizie, $paginaHTML);
-
+                exit;
```

Comment:
A cosa serve questo exit?

Reply:
Per fixare l'errore che veniva fuori.
Non ho trovato altro modo
Questo perché dopo un echo non puoi usare header (anche se in teoria non ci andava visto che era in un branch diverso)